study.md

```diff
@@ -27,17 +27,23 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+The HTML that I write.
+'View Source' in my browser (which is essentially the same as the HTML that I write)
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes. In part because my Dev Tools will correct some errors in my HTML, making it different from
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.